client/src/components/main/chatwindow/DeleteComment.js

```diff
@@ -0,0 +1,148 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import Styled from './styles/comment.styles';
+import axios from 'axios';
+import { ChatContext } from './ChatWindow';
+
+const DeleteComment = props => {
+  const {
+    id,
+    name,
+    date,
+    text,
+    user_id,
+    userImage,
+    isEdited,
+    deleteComment,
+    setDeleteComment,
+    useAvatar,
+  } = props;
+  const { dispatch } = useContext(ChatContext);
+  const deletedId = 'deletedId';
+
+  const handleDelete = e => {
+    axios
+      .delete(`/api/comments/${id}`, {
+        headers: { authorization: `bearer ${localStorage.authToken}` },
+      })
+      .then(() => {
+        dispatch({ type: 'DELETE_FROM_DB', text });
+      })
+      .catch(err => console.error(err));
+
+    setDeleteComment(false);
+  };
+
+  useAvatar(user_id, userImage, deletedId);
+
+  return (
+    <div>
+      <DeletePopout>
+        <Title>Delete Message</Title>
+        <Message>Are you sure you want to delete this comment? This cannot be undone.</Message>
+        <CommentContainer>
+          <Styled.CommentAvatar id={deletedId}></Styled.CommentAvatar>
+          <Styled.CommentNameDateWrapper>
+            <Styled.CommentName>{name}</Styled.CommentName>
+            <Styled.CommentTime>{date}</Styled.CommentTime>
+          </Styled.CommentNameDateWrapper>
+          <CommentText>
+            <Styled.CommentText>{text}</Styled.CommentText>
+            <Styled.CommentEdited isEdited={isEdited}>(edited)</Styled.CommentEdited>
+          </CommentText>
+        </CommentContainer>
+        <ButtonWrapper>
+          <Styled.Button onClick={() => setDeleteComment(false)}>Cancel</Styled.Button>
+          <Styled.DeleteButton onClick={handleDelete}>Delete</Styled.DeleteButton>
+        </ButtonWrapper>
+      </DeletePopout>
+      {deleteComment && <Overlay></Overlay>}
+    </div>
```

Comment:
Can we get some empty lines in there to help make the code more visually appealing?

Reply:
Should I add an empty line between the outer level components?

Reply:
I'll have to see it to understand it